tests/cypress/e2e/config.cy.js

```diff
@@ -22,4 +22,12 @@ describe('Config', () => {
   runConfigTest('showNewCollectionButton=false', 'showNewCollectionButton false', () => {
     cy.get('[data-cy="new-collection"]').should('be.visible', false)
   });
+  runConfigTest('', 'translations: No input for `lang` and `translations` object -> read from default `en` file', () => {
```

Comment:
this is a default test, you can move it inside the first runConfigTest funtion, so we run all default config tests in one function